gateway-ha/src/main/java/io/trino/gateway/ha/security/LbFormAuthManager.java

```diff
@@ -48,7 +50,11 @@ public LbFormAuthManager(FormAuthConfiguration configuration,
             Map<String, String> pagePermissions)
     {
         this.presetUsers = presetUsers;
-        this.pagePermissions = pagePermissions;
+        this.pagePermissions = pagePermissions.entrySet().stream()
+                .filter(entry -> entry.getKey() != null && entry.getValue() != null)
```

Comment:
When do we receive null keys?

Reply:
Please throw an exception if it doesn't happen in a real situation.

Reply:
Sure, i added try-catch NPE when doing UpperCase for key.

Reply:
No need to add try-catch block in my opinion. I would just throw an exception.

Reply:
Reminder.

Reply:
I pushed changes instead.

Reply:
Missed it. removed try-catch